### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NiccoMlt**
I updated the README file explaining how to install Xtext and PMD in Eclipse and how to install Eclipse using ubuntu-make

### CODE REVIEW

**Comment by DanySK** [comment]
Please use better commit messages from now on.

File: README.md [hunk 1]
```diff
@@ -108,6 +123,7 @@ The project has been developed using Eclipse, and can be easily imported in such
   * If you are a Windows user: `gradlew.bat alchemist-incarnation-biochemistry:generateGrammarSource`
 
   Go to Eclipse, right click on `alchemist-incarnation-biochemistry` project -> Refresh
+* If you have errors in `alchemist-projectview` project, make sure Xtend plugin is correctly installed
```
Comment (DanySK) [inline README.md]: Please expand on that, suggest the installation of the Xtext plugin from the market instead (it's required anyway)

Comment (NiccoMlt) [inline README.md]: I already explained it in the "Recommended configuration" part (rows 72-74 of the updated README.md).
I have to copy/move that after that "troubleshooting" part (row 126)?

Comment (DanySK) [inline README.md]: Whoops, I oversaw that. Just change xtend to Xtext in the last sentence.

**Pull request merged**